app/Components/Statistics/TotalStatistics.tsx

```diff
@@ -20,6 +27,25 @@ const TotalStatistics = (props: TotalStatisticsProps) => {
 
   const theme = useTheme();
 
+  // sort by most number of hints
+  props.numHintsUsedPerStrategy.sort((a, b) => {
+    return b.numHintsUsed - a.numHintsUsed;
+  });
+
+  const strategyHints: React.JSX.Element[] = [];
+  for (const strategyHint of props.numHintsUsedPerStrategy) {
+    strategyHints.push(
+      <Statistic
+        statisticName={
+          "  " + formatOneLessonName(strategyHint.hintStrategy) + ": "
+        }
+        statisticValue={strategyHint.numHintsUsed}
+        testID={"hintsUsed" + strategyHint.hintStrategy}
+        key={strategyHint.hintStrategy}
+      />
+    );
+  }
```

Comment:
I have pushed up a change to avoid the mutation of props. I prefer using the for of approach instead of map. The arbitrary spacing is so that the elements are indented when rendered in the site.